go/src/meguca/server/router.go

```diff
@@ -249,20 +249,34 @@ func healthCheck(w http.ResponseWriter, r *http.Request) {
 }
 
 // Get YouTube title and googlevideo URL from URL
-func getYouTubeData(w http.ResponseWriter, r *http.Request) {
-	info, err := ytdl.GetVideoInfo(("https://www.youtube.com/watch?v=" + extractParam(r, "id")))
+func youTubeData(w http.ResponseWriter, r *http.Request) {
+	info, err := ytdl.GetVideoInfo("https://www.youtube.com/watch?v=" + extractParam(r, "id"))
 
 	if err != nil {
 		httpError(w, r, err)
 		return
 	}
 
-	video, err := info.GetDownloadURL(info.Formats.Best("mp4")[0])
+	video, err := info.GetDownloadURL(info.Formats.
+		Filter(ytdl.FormatExtensionKey, []interface{}{"webm"}).
+		Filter(ytdl.FormatResolutionKey, []interface{}{"360p"})[0])
```

Comment:
This will panic, if for some reason nothing is found. Needs a bound check and:  
ezmode: should return a HTTP error to the client
harder mode: should reselect a different resolution